### PR TITLE
remove platform + put more to nightlies

### PR DESCRIPTION
**Pull request by ebarboni**
This is an update to remove the generation of platform.

+ 
vsix to nightlies
mavenrepo zipped to nightly
installers to nightlies

if it's ok to merge it would be nice to force a build to test :D.

### CODE REVIEW

File: vars/asfMainNetBeansBuild.groovy [hunk 1]
```diff
@@ -392,23 +396,26 @@ def doParallelClusters(cconfigs) {
...
+                                        zip zipFile:'mavenrepo.zip',dir:'mavenrepository',archive:'false'
+                                        //archiveArtifacts 'mavenrepository/**'
+                                        publishToNightlies("/netbeans/candidate/mavenrepository","mavenrepo.zip")
+                                    }    
+                                    // make vsix available to dist to pickup (only for main release) need a maven setup
+                                    sh "ant -f build-${clustername}-temp/java/java.lsp.server build-vscode-ext -Dvsix.version=${vsixversion} -Dmetabuild.branch=${branch}"
+                                    //sh "cp -r build-${clustername}-temp/java/java.lsp.server/build/*.vsix dist/vsix/"
```
Comment (neilcsmith-net) [inline vars/asfMainNetBeansBuild.groovy:417]: Isn't that line needed to archive vsix on build?

**Review by neilcsmith-net — CHANGES_REQUESTED** [review]
Removing platform is good.  Publishing vsix (and possibly Windows installer) to nightlies good.

I would keep all archiving on builds.  Nightlies doesn't have zip download for all release artefacts.  Also, can't keep forever the right build.

I would actually stop publishing NBMs to nightlies, for now at least.  No use unless we can rework the update centre redirects.  And I'm not sure about uploading Maven.  Takes a lot of time.  Why do we need it there?

**Comment by ebarboni** [comment]
For maven I would try to get only the zip, as storing artefacts take long time.
keep all to dist. Noted the javadoc for removal later, trying to remove nbm also.

**Comment by ebarboni** [comment]
@neilcsmith-net not sure we are ok there. 
change isthe zip only maven repo instead of 1go of artifacts we do not use. If I remember well

**Comment by neilcsmith-net** [comment]
Zip only maven repo _on ci-builds_ good with me.  I just download "all files in zip" there anyway.  Just should not be on nightlies - can't be locked, gets overwritten, etc.  As long as we archive all artefacts on ci-builds that are pushed to nightlies I'm good with the changes.  I don't think we should use publish to nightlies as a replacement for the archived artefacts, just as the download source for end users.

**Comment by ebarboni** [comment]
you're right, merging this for now.

**Pull request merged**